src/index.ts

```diff
@@ -24,7 +24,9 @@ import axios from 'axios'
 import GraphSource from './providers/subgraphProviderWrapper'
 import Web3Source from './providers/web3ProviderWrapper'
 import EthereumSource from './providers/ethereumProviderWrapper'
-import { utils } from './lib/utils'
+import {
+    decodeAllData, decodeKey, decodeKeyValue, encodeAllData, encodeArrayKey, encodeKey, getSchemaElement
+} from './lib/utils'
```

Comment:
Isn't the libs just an internal library? I think selecting here functions doesnt improve a lot. External dependencies are the problem

Reply:
I assumed the utils would also be imported directly from time to time. Either way it doesnt hurt imo.

I'm setting up a little project to investigate the size properly.